tests/test_finite_difference.py

```diff
@@ -119,10 +119,11 @@ def test_ODE_stdev():
     a = -k * x
 
     model = sf.ODEModel({
-        sf.D(v, t): a,
-        sf.D(x, t): v,
-    },
-        initial={v: 0, x: 1, t: 0})
+            sf.D(v, t): a,
+            sf.D(x, t): v,
+        },
+        initial={v: 0, x: 1, t: 0}
+    )
```

Comment:
If you want to go full PEP-8 it's still wrong. Either indent `sf.D` and `}` to align with `{`, or put the opening `{` on a separate line at the same indentation `},` and `initial` are now.